api/src/main/java/org/ihtsdo/buildcloud/controller/ProductController.java

```diff
@@ -48,7 +48,7 @@
 
 	public static final String[] PRODUCT_LINKS = {"manifest", "inputfiles","sourcefiles","builds","buildLogs"};
 
-	@RequestMapping( method = RequestMethod.GET )
```

Comment:
@Tyler-Cooksey routinely updating _some_ controllers in our applications to use `GetMapping`, `PostMapping` rather than `RequestMapping` is just making our code inconsistent. The is nothing wrong with `RequestMapping` and that is what most of us are used to but if you have a strong argument for converting everything to the other style please do it consistently for all controllers within an application, maybe in a separate commit. Thanks

Reply:
Thank you for your comment and the main reason for changing this is that I like to stay up to date with the latest annotations available inside Spring and these are nice wrappers around the REST interfaces we use so in my opinion makes sense to use them instead as it makes the code less verbose. I will update all the other controllers to follow the same style also.